intrinsicsolver/fixFunctions.go

```diff
@@ -34,27 +38,41 @@ func FixFunctions(template []byte, logger *logger.Logger) ([]byte, error) {
 		quotationProcessed = append(quotationProcessed, fixed)
 	}
 
+	// In case the intrinsic function is in the last line and the the next line is investigated in search for it's multi-line body, we have to add one, blank line.
+	quotationProcessed = append(quotationProcessed, "")
+
 	lines := quotationProcessed
 
-	// These are the YAML short names of a functions which take the arguments in a form of a map.
-	multiLiners := []string{"!FindInMap", "!Join", "!Select", "!Split", "!Sub", "!And", "!Equals", "!If", "!Not", "!Or"}
+	var functions = []string{"Base64", "GetAtt", "GetAZs", "ImportValue", "Ref", "FindInMap", "Join", "Select", "Split", "Sub", "And", "Equals", "If", "Not", "Or"}
 
 	for idx, d := range lines {
-		for _, function := range multiLiners {
-			fixMultiLineMap(&d, &lines, idx, function)
+		for _, m := range mode {
+			if m == "multiline" {
+				for _, function := range functions[5:] {
```

Comment:
Let's save `functions[5:]` into some intermediate variable with a meaningful name (that would describe why we're slicing from the 5th position).

Reply:
Ok